test/docker-compose.yml

```diff
@@ -415,15 +415,17 @@ services:
 
   zkevm-mock-l1-network:
     container_name: zkevm-mock-l1-network
-    image: hermeznetwork/geth-zkevm-contracts:v1.1.0-fork.4
+    image: hermeznetwork/geth-zkevm-contracts:v1.1.0-snapchain
     ports:
       - 8545:8545
       - 8546:8546
     environment:
       - VIRTUAL_HOST=rpc-l1.<user-rollup-identifier>
       - LETSENCRYPT_HOST=rpc-l1.<user-rollup-identifier>
       - VIRTUAL_PORT=8545
-    command:
+      - L2_CHAIN_ID=<l2-chain-id>
```

Comment:
likely we also need to change initialZkEVMDeployerOwner/zkEVMDeployerAddress/networkName in zkevm-contracts/docker/scripts/deploy_parameters_docker.json. so we need to think of a better way to pass in those params instead of using `sed` command. so maybe pass in a .json config file and merge w deploy_parameters_docker.json?